conans/test/integration/tools/system/package_manager_test.py

```diff
@@ -229,11 +229,51 @@ def fake_check(*args, **kwargs):
             return ["package1", "package2"]
         from conan.tools.system.package_manager import _SystemPackageManagerTool
         with patch.object(_SystemPackageManagerTool, 'check', MagicMock(side_effect=fake_check)):
-            tool.install(["package1", "package2"])
+            tool.install(["package1", "package2"], host_package=True)
```

Comment:
```suggestion
            tool.install(["package1", "package2"])
```
Default not necessary